examples/src/main/scala/sttp/tapir/examples/BooksExample.scala

```diff
@@ -11,7 +11,7 @@ object BooksExample extends App with StrictLogging {
   case class Author(name: String, country: Country)
   case class Genre(name: String, description: String)
   case class Book(title: String, genre: Genre, year: Int, author: Author)
-  case class BooksQuery(genre: Option[String], limit: Limit)
+  case class BooksQuery(genre: Option[String], limit: Limit, hiddenParameter: Option[String])
```

Comment:
I think I'd prefer to keep this example as simple as possible - hidden, while useful, isn't a core feature we want to showcase when you first encounter tapir :)